src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,22 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, Bishan Ave 5, #15-12, 460123";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in BLOCK, STREET, UNIT, POSTAL_CODE format";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
+    public static final String SPLIT_TOKEN = ", ";
+
+    public final int BLOCK_INDEX = 0;
```

Comment:
2 things:
- Good that you have eliminated magic numbers in the code.
However,
- Please check the correct convention for naming constants.